pyproject.toml

```diff
@@ -69,9 +69,9 @@ include = ["/src"]
 [tool.mypy]
 python_version = "3.12"
 
-[tool.ruff]
-line-length = 88
-select = ["E", "F", "I", "C"]
+[tool.ruff.lint]
+select = ["E", "F", "I", "C", "TCH", "FA", "UP"]
```

Comment:
Nice, this is great!
Since you introduced "UP", I think it's important to set [target-version](https://docs.astral.sh/ruff/settings/#target-version) as well since the library should still run with 3.7 and I think this created some invalid behavior without setting this version.

Reply:
Added in https://github.com/CodSpeedHQ/pytest-codspeed/pull/25/commits/887ad44574655d5662487c8c1e21adaa7fb8da21